components/dataViews/TransactionInfo/TxMsgTransferDetails.tsx

```diff
@@ -12,7 +12,7 @@ const TxMsgTransferDetails = ({ msgValue }: TxMsgTransferDetailsProps) => {
     msgValue.token,
     "Token must be set, same as https://github.com/osmosis-labs/telescope/issues/386",
   );
-  const timeoutDateObj = new Date(msgValue.timeoutTimestamp.divide(1_000_000).toNumber());
+  const timeoutDateObj = new Date(msgValue.timeoutTimestamp.multiply(1_000_000).toNumber());
```

Comment:
In the MsgTransfer instance this is in nanoseconds. So here the old divide was correct to go to milliseconds.

```
  /**
   * Timeout timestamp in absolute nanoseconds since unix epoch.
   * The timeout is disabled when set to 0.
   */
  timeoutTimestamp: Long;
```